mdbook/src/chapter_4/chapter_4_2.md

```diff
@@ -135,11 +137,11 @@ Second, just before a worker might hypothetically receive a message at `(l1, t1)
 
 At this point, we have two important bits of knowledge:
 
-1. The accumulated values for all pointstamps less than `(l1, t1)` were initially non-positive, by assumption.
+1. The accumulated values for all pointstamps less than `(l1, t1)` were initially non-positive, by assumption.[because everything is settled except the inflight message, which means someone has decremented for it, but the receiver hasn't incremented?]
```

Comment:
No, this "by assumption" is meant to be "by the assumption of the safety property". Perhaps it is too tangled up in the text, but the claim is that "if there are no pointstamps with positive count that could-result-in `(l1,t1)`, then ..."